src/dmd/frontend.h

```diff
@@ -12497,7 +12497,10 @@ struct Global
     DArray< char > written;
     Array<const char*>* path;
     Array<const char*>* filePath;
-    DArray< char > _version;
+    ENUM_CONSTANT(const char*, _version, "v2.093.1-503-g4c81d4e38")
+
+    ENUM_CONSTANT_NUMERIC(uint32_t, _versionNumber, 2093u)
```

Comment:
do you plan to leave this hardcoded?

Reply:
See my prior comment - https://issues.dlang.org/show_bug.cgi?id=21217 and #11679.

Reply:
> leave this hardcoded?

It's gone now ;-)